README.md

```diff
@@ -74,6 +74,12 @@ To check the SDK is setup correctly, run the following in a bash console:
 $ python -m vortexasdk.check_setup
 ```
 
+or
+
+```bash
+$ python3 -m vortexasdk.check_setup
```

Comment:
i'm shit at python and am sure there's a smart symlink thing here, but on most modern macs this will work and just `python` will not